src/controllers/login.js

```diff
@@ -0,0 +1,33 @@
+const bcrypt = require('bcryptjs');
+const { sign } = require('jsonwebtoken');
+const loginQuery = require('../database/queries/login');
+
+exports.get = (request, response) => {
+  response.render('login', { layout: 'login' });
+};
+
+exports.post = (request, response) => {
+  const { usernameValue } = request.body;
+  const { passwordValue } = request.body;
+  loginQuery(usernameValue, (error, result) => {
+    if (error) return response.send(JSON.stringify({ err: 'error in login !' }));
```

Comment:
This one is also about the error message... So it ain't a big deal.
You're a server that is sending data to a professional front end, you're not sending it to a user whom you should try to simplify things for as possible, so it would have a false context to say `error in login !`, and should be correct to say `login_error`. This applies to all of your responses to the server.

You're free to change or ignore this comment. @HemaSAli .

Reply:
thanks nour , I did it .